src/borg/threading/create.py

```diff
@@ -624,6 +624,13 @@ def pack_and_add_item(self):
         self.num_items += 1
         if 'chunks' in item:
             self.archive.stats.nfiles += 1
+        try:
+            # XXX This is the sledgehammer approach of getting rid of these, see comment in Item
+            del item.original_path
+            del item.status
+            del item.num_chunks
+        except AttributeError:
+            pass
```

Comment:
These are added when an item was processed by the pipeline, then all three should be present; if the item was not processed by the pipeline, none should be present.

I want an overall nicer way for this, though, that is safer to use and makes it more clear which attributes are runtime only and which are actually stored (see item.pyx).